packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -416,16 +417,25 @@ export class AmqpConnection {
       exchange,
       routingKey,
       createQueueIfNotExists = true,
+      assertQueueErrorHandler = defaultAssertQueueErrorHandler,
     } = subscriptionOptions;
 
     let actualQueue: string;
 
     if (createQueueIfNotExists) {
-      const { queue } = await channel.assertQueue(
-        subscriptionOptions.queue || '',
-        subscriptionOptions.queueOptions || undefined
-      );
-      actualQueue = queue;
+      const queueName = subscriptionOptions.queue || '';
+      const queueOptions = subscriptionOptions.queueOptions || undefined;
```

Comment:
If `queueOptions` is not defined its going to be undefined so using `||` operator is not necessary

Reply:
should be good